package.json

```diff
@@ -49,7 +49,7 @@
     "is-nan": "^1.2.1",
     "memory-cache": "^0.1.6",
     "moment": "^2.14.1",
-    "moment-opening-times": "git://github.com/nhsuk/moment-opening-times.git",
+    "moment-opening-times": "git://github.com/nhsuk/moment-opening-times.git#feature\/performance-improvements",
```

Comment:
This just needs changing to: `"moment-opening-times": "git://github.com/nhsuk/moment-opening-times.git#2.0.0",` and it is all good to merge.